includes/compatibility/class-wcpdf-compatibility-third-party-plugins.php

```diff
@@ -317,6 +320,39 @@ function restore_wgm_thumbnails( $document_type, $document ) {
 			add_filter( 'woocommerce_order_item_name', array( 'WGM_Product', 'add_thumbnail_to_order' ), 100, 3 );
 		}
 	}
+
+	/**
+	 * Adds invoice number filter to the search filters available in the admin order search.
+	 * 
+	 * @param array $options List of available filters.
+	 * 
+	 * @return array
+	 */
+	function hpos_admin_search_filters( array $options ): array {
+		unset( $options['all'] );
+		$options['invoice_number'] = __( 'Invoice number', 'woocommerce-pdf-invoices-packing-slips' );
+		$options['all']            = __( 'All', 'woocommerce-pdf-invoices-packing-slips' );
```

Comment:
Maybe it would be better if you stored the "all" option and then brought it back here. In this case, if any changes happen to the "All" text (which is unlikely!), it'll not override that. 
I mean something like this:
```php
$all = $options['all'];
unset( $options['all'] );
$options['invoice_number'] = __( 'Invoice number', 'woocommerce-pdf-invoices-packing-slips' );
$options['all'] = $all;

```

Also, you used our text domain, which is different from the Woo.

What do you think?

Reply:
@MohamadNateqi I agree, with this approach we remove the possibility of some changes made by WooCoommerce that are not followed by our code.